oak_containers_stage1/src/main.rs

```diff
@@ -19,17 +19,18 @@
 mod client;
 mod image;
 
+use anyhow::Context;
 use clap::Parser;
 use client::LauncherClient;
 use image::Image;
 use std::error::Error;
 
 #[derive(Parser, Debug)]
 struct Args {
-    #[arg(long, required = true)]
+    #[arg(long, default_value_t = 2)]
```

Comment:
This could possibly even be a constant, if we assume the launcher always run in the host rather than another VM.